vignettes/response-FDA IR-pilot3.qmd

```diff
@@ -0,0 +1,243 @@
+---
+title: |
+   | R Consortium R Submission Working Group
+   | Pilot 3
+   |
+   | Health Authority Request
+   |
+subtitle: "\nResponse to FDA Information Request"
+format: 
+  pdf:
+    toc: false
+    number-sections: true
+    include-before:
+      - '\newpage{}'
+editor: source
+---
+
+{{< pagebreak >}}
+
+![](images/RConsortium_Horizontal_Pantone.png){width="228"}
```

Comment:
This image is missing?

Reply:
It seems so, @laxamanaj can we add this image?

Reply:
Apologies.  I have pushed the images to the remote branch.  It was still in my local.  When you re-render now it should look like this : 
![image](https://github.com/RConsortium/submissions-pilot3-adam/assets/95468392/35263e3f-6fd1-48e3-a21d-c773735bfb1a)